docs/updates/2019-09.md

```diff
@@ -0,0 +1,116 @@
+(app, object, connection, ...). Instead of returning the whole table the command will return only the ids.
+
+```bash
+$ corectl connection ls --quiet
+3d196c6e-cc87-4f0b-a4ab-48dabb9ffed7
+f39601c0-70be-4e48-9a75-4085e69262fc
+3718abde-ef0e-438b-8941-bd3185c33f6c
+9c632be3-f844-466a-8546-3b56291215dc
+```
+
+This enables scripts like piping ids back to corectl in order to do multiple commands. As an example removing all
+connections in an app can be done by:
+
+```bash
+$ corectl connection ls --quiet | xargs corectl connection rm
+Saving app...
+App successfully saved
+
+$ corectl connection ls
++----+------+------+
+| Id | Name | Type |
++----+------+------+
++----+------+------+
+```
+
+### Introducing context
+
+Contexts store connection information such as engine url, certificates and headers, similar to a config. The main
+difference between contexts and configs is that they can be used globally. Using the context you can quickly switch
+between different environments such as local, stage, production, Qlik Core and Qlik Sense Enterprise.
+
+Use the context sub-commands `clear`, `get`, `ls`, `rm`, `set`, `use` to work with the context. Note that contexts have
+the lowest precedence. This means that e.g. an --engine flag (or an engine field in a config) will override the engine
+url in the current context.
+
+As an example you can setup two contexts:
+
+```bash
+corectl context set testapps --engine localhost:39076 --comment "All my local test apps"
+corectl context set deployment --engine wss://<qlik-sense-url>:4747 --certificates <certfolder> --headers "X-Qlik-User: UserDirectory=<userdir>; UserId=<userid>"
+corectl context ls
+```
+
+and switch between them using while using the same load script file.
+
+```bash
+corectl context use testapps
+corectl script set script.qvs --app test
+corectl reload
+
+corectl context use deployment
+corectl script set script.qvs --app deployedapp
+corectl reload
+```
+
+### Support for alternate states
+
+If you have been working with Qlik for a while you might have come across alternate state. The feature allows you to
+make different selections on the same dimension, and compare the selections in a single visualization or in two or more
+visualizations side by side. You can also use it in set analysis in order to compare expressions with different
+selections.
+
+In corectl you manipulate the alternate states in the `state` command with the following actions `add`, `ls` and `rm`.
+
+### Other fixes
+
+Together with the larger changes we also have the following improvements:
+
+* Added `--limit` flag to reload to limit number of rows to load
+* Performance improvements when creating objects
+* Sort ids alphanumerically
+* Fixed URL escaping in json files to improve readability of expressions.
+
+## Changes in catwalk
+
+* catwalk: Added full title when hoovering over an application in the document list.
+* catwalk: Added information about supported browsers. Firefox is now fully supported.
```

Comment:
I think we should skip mentioning these catwalk changes.. they are really small, and looks like github commit messages. We don't mention any other of our small repo fixes..